lib/new_relic/agent/instrumentation/sinatra/chain.rb

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'new_relic/agent/instrumentation/controller_instrumentation'
+require 'new_relic/agent/instrumentation/sinatra/transaction_namer'
+require 'new_relic/agent/instrumentation/sinatra/ignorer'
+require 'new_relic/agent/parameter_filtering'
+
+require_relative 'instrumentation'
+
+module NewRelic::Agent::Instrumentation
+  module SinatraInstrumentation
```

Comment:
rename `SinatraInstrumentation` to just `Sinatra` to match with other instrumentations where we're doing the same.